github/github.md

```diff
@@ -0,0 +1,130 @@
+## GitHub darslari
```

Comment:
bunga gitga kirish darsni qo'shsak ajoyib bo'ladi!

Reply:
Lekin bu faylni bu branchdan olib tashlasak ham bo'ladi

Reply:
to'g'irlandi